e2e/utils/logout.ts

```diff
@@ -0,0 +1,15 @@
+import { Page, expect } from '@playwright/test';
+import translations from '../../client/i18n/locales/english/translations.json';
+
+export async function signout(page: Page) {
+  await page.goto('/');
+  await page.getByRole('button', { name: translations.buttons.menu }).click();
+  await page
+    .getByRole('button', { name: translations.buttons['sign-out'] })
+    .click();
+
+  await page
+    .getByRole('button', { name: translations.signout.certain })
+    .click();
+  await expect(page).toHaveURL(/.*\/learn\/?$/);
```

Comment:
It might be better to check that the sign-in button has re-appeared, rather than the redirect, since the sign-in button is only visible if you're signed out.